### PR TITLE
Add ScheduledQueryRun resource

### PR DESCRIPTION
**Pull request by remi-stripe**
r? @jlomas-stripe @brandur-stripe 
cc @stripe/api-libraries

### CODE REVIEW

**Comment by jlomas-stripe** [comment]
Looks awesome to me.

File: lib/stripe.js [hunk 1]
```diff
@@ -83,6 +83,9 @@ var resources = {
     Transactions: require('./resources/Issuing/Transactions'),
     Disputes: require('./resources/Issuing/Disputes'),
   }),
+  Sigma: resourceNamespace('sigma', {
+    ScheduledQueryRuns: require('./resources/Sigma/ScheduledQueryRun'),
```
Comment (brandur-stripe) [inline lib/stripe.js]: It's kind of annoying, but I think the convention in Node is to have the filenames plural. Can we make this `ScheduledQueryRuns`?

**Comment by brandur-stripe** [comment]
Minor comment, but looks good!

File: lib/stripe.js (hunk 1 above)
Comment (remi-stripe) [reply lib/stripe.js]: ah nice catch on that one. I just fixed it if you can take a second look

**Comment by brandur-stripe** [comment]
Thanks for the fixes! LGTM.

**Comment by brandur-stripe** [comment]
Released as 6.5.0.

**Pull request merged**